src/app/widgets/svg-wind/svg-wind.component.svg

```diff
@@ -1,14 +1,13 @@
 <svg
-   width="100%"
-   height="100%"
-   viewBox="0 0 462 462"
-   version="1.1"
-   id="svg8"
-   xmlns:xlink="http://www.w3.org/1999/xlink"
-   xmlns="http://www.w3.org/2000/svg"
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-   xmlns:cc="http://creativecommons.org/ns#">
+  viewBox="0 0 462 462"
+  preserveAspectRatio="xMidYMid"ù
```

Comment:
I know this PR has been merged, but that extra character at the end of the line looks wrong.

Reply:
Hi Greg. Good catch. Thanks!

I'm curious to know What's your interest in going over old commits?